docgen/content-sources/v1/toc.yaml

```diff
@@ -1,148 +1,148 @@
 toc:
   - title: 'functions'
-    path: /docs/reference/functions/v1_cloud_functions_.html
+    path: /docs/reference/functions/cloud_functions.html
```

Comment:
Filenames currently live on firesite still use the underscores, such as cloud_functions_#cloudfunction and config_.config.

If we're 100% sure the script will generate the files without underscore naming, this should work fine.

Reply:
This was modified to match the output of the gen script.